main/lsp/notifications/sorbet_workspace_edit.h

```diff
@@ -15,6 +15,8 @@ class SorbetWorkspaceEditTask final : public LSPDangerousTypecheckerTask {
     // Caches the fast path decision for the provided update. Becomes invalidated when the update changes.
     mutable bool cachedFastPathDecisionValid = false;
     mutable bool cachedFastPathDecision = false;
+    // HACK: In the event that this edit is too large to index serially, stash the indexer here for use in `runSpecial`.
+    LSPIndexer *indexer = nullptr;
```

Comment:
Are there lifetime problems here? Can the pointed to thing have been freed by the time the edit task wakes up and is processed?

Reply:
Nope. The thread that could do so is parked until the notification is hit.